tests/test_mlkit.py

```diff
@@ -277,7 +277,11 @@
 PAGE_SIZE_VALUE_ERROR_MSG = 'Page size must be a positive integer between ' \
                             '1 and {0}'.format(mlkit._MAX_PAGE_SIZE)
 INVALID_STRING_OR_NONE_ARGS = [0, -1, 4.2, 0x10, False, list(), dict()]
-
+PUBLISH_AND_UNPUBLISH_VALUE_ARGS = [
```

Comment:
PUBLISH_AND_UNPUBLISH_FUNCS_WITH_ARGS

Reply:
pylint doesn't like more than 30 characters. Changed to PUBLISH_UNPUBLISH_WITH_ARGS